python/pyspark/sql/tests/test_udtf.py

```diff
@@ -2482,6 +2533,7 @@ def tearDownClass(cls):
             super(UDTFTests, cls).tearDownClass()
 
 
+'''
```

Comment:
Looks like a mistake? Could you revert this change?

Reply:
My mistake on this, reverted.